### ISSUE TITLE
WHERE "NOT", "OR", "AND"

### ISSUE THREAD

**Issue by v1a0** [v1a0/sqllex#1]
WHERE currently no support "<" or ">" or "NOT" or "OR" or "AND" and other things except "="

**Comment by V1A0** [comment]
## Where support "<", '<<', '<=', '>=', '>>', '>', '=', '==', '!=', '<>' added since Sqllex version 0.1.6

```python

db.select(
    TABLE='users',
    WHERE={
        'age': 10,    # where age = 10
    },
    ORDER_BY='username'
)

db.select(
    TABLE='users',
    WHERE=['age', 10],    # where age = 10
    ORDER_BY='username'
)

db.select(
    TABLE='users',
    WHERE=['age', [10, 11]],    # where age = 10 and 11, wait a sec... i guess i fucked up...
    ORDER_BY='username'
)

db.select(
    TABLE='users',
    WHERE=['age', '!=', 10],    # where age != 10
    ORDER_BY='username'
)

db.select(
    TABLE='users',
    WHERE={
        'age': 10,    # where age = 10
        'group': ['<', 10]    # and where group < 10
    },
    ORDER_BY='username'
)

db.select(
    TABLE='users',
    WHERE={
        'age': 10,    # where age = 10
        'group': ['!=', 5, 10]    # where group != 5 and 10
    },
    ORDER_BY='username'
)
```

**Comment by v1a0** [comment]
# Fixed since Sqllex version 0.1.9.8

**Comment by v1a0** [comment]
# Fixed since Sqllex version 0.1.7.2
